src/libraries/System.Collections/tests/Generic/PriorityQueue/PriorityQueue.Generic.Tests.cs

```diff
@@ -34,6 +34,28 @@ protected PriorityQueue<TElement, TPriority> CreatePriorityQueue(
             return queue;
         }
 
+        [Theory]
+        [InlineData(1)]
+        [InlineData(100)]
+        public void CreateWithCapacity_EqualsCapacityProperty(int capacity)
+        {
+            var queue = new PriorityQueue<TElement, TPriority>(capacity);
+            Assert.Equal(capacity, queue.Capacity);
+        }
+
+        [Theory]
+        public void PriorityQueue_EnsureCapacityThenTrimExcess_CapacityUpdates()
+        {
+            var queue = new PriorityQueue<TElement, TPriority>(2);
+            Assert.Equal(2, queue.Capacity);
+
+            queue.EnsureCapacity(12);
+            Assert.True(queue.Capacity >= 12, "capacity should be at least 12 after ensuring capacity");
```

Comment:
```suggestion
            Assert.InRange(queue.Capacity, 12, int.MaxValue);
```

Reply:
Is this a common way to express such assertions? The xunit author recommends using `Assert.True`

Reply:
> Is this a common way to express such assertions?

Yes

> The xunit author recommends using Assert.True

Where? Why?

Reply:
https://github.com/xunit/xunit/discussions/2143#discussioncomment-75557

Reply:
I guess I'm somewhat put off `InRange` because you need to specify an artifical upper bound. I would have reached for `Assert.GreaterThanOrEqual` if it was available.

Reply:
> [xunit/xunit#2143 (comment)](https://github.com/xunit/xunit/discussions/2143#discussioncomment-75557)

I think that's in the eye of the beholder. You get the message for free with InRange, and it very naturally encapsulates exactly what's being verified.  The upper bound isn't really artificial; that is the desired range.  It reads much better to me, and leads to more consistency when asserts do fail. Not going to die on this hill, though.

Reply:
I suppose the appeal of `Assert.True` would have been stronger assuming the message parameter had been annotated with `CallerArgumentExpressionAttribute`. Guessing that's not on the cards for xunit 2 though.